src/main/java/com/microsoft/azure/functions/annotation/TwilioSmsOutput.java

```diff
@@ -12,23 +12,63 @@
 import java.lang.annotation.Target;
 
 /**
+ * <p>Place this on a parameter whose value would be written to twilio SMS.
+ * The parameter type should be OutputBinding&lt;T&gt;, where T could be one of:</p>
+ *
+ * <ul>
+ *     <li>Any native Java types such as int, String, byte[]</li>
+ *     <li>Any POJO type</li>
+ * </ul>
+ *
  *
  * @since 1.0.0
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.PARAMETER, ElementType.METHOD})
 public @interface TwilioSmsOutput {
+    /**
+     * The variable name used in function.json.
+     * @return The variable name used in function.json.
+     */
     String name();
 
+    /**
+     * <p>Defines how Functions runtime should treat the parameter value. Possible values are:</p>
+     * <ul>
+     *     <li>"" or string: treat it as a string whose value is serialized from the parameter</li>
```

Comment:
Why it's inconsistent with TimerTrigger.java even the declaration is the same?

Reply:
The definition is slightly different between Trigger, Input and Output.